src/liballoc/arc.rs

```diff
@@ -251,7 +253,9 @@ impl<T: ?Sized> Arc<T> {
             let cur = this.inner().weak.load(Relaxed);
 
             // check if the weak counter is currently "locked"; if so, spin.
-            if cur == usize::MAX { continue }
+            if cur == usize::MAX {
+                continue
```

Comment:
I personally prefer to leave off `;` whenever it's a terminating statement that can't have anything following it anyway (e.g. `break`, `return`, `continue`) but that may be just my style